src/pages/reset-password/index.tsx

```diff
@@ -35,19 +35,8 @@ const StyledButton = styled(Button)`
     margin: 0.8rem 0.4rem;
 `
 
-const resetValidation = (values: EmailType) => {
-    const errors: ErrorType = {}
-    const email = trimSpaces(values.email)
-    const email_error = validation.required(email) || validation.email(email)
-
-    if (email_error) {
-        errors.email = email_error
-    }
-
-    return errors
-}
-
 const ResetPassword = () => {
+    const [apiError, setApiError] = useState<string | null>('')
     const initialValues: EmailType = { email: '' }
```

Comment:
If i am not wrong,do we need this const `initialValues`?

Reply:
Hi @aswathy-deriv thanks for highlighting, but we are using this on line number `109`. 🙏